CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
```

Comment:
Do we need this YAML header? I thought that `CONTRIBUTING.md` is like `README.md` or `FAQ.md`.